src/components/redux/actions/index.jsx

```diff
@@ -1,8 +1,88 @@
-import { FIRST_TYPE } from "./actionsTypes";
+import {
+  ADD_TO_CART,
+  DELETE_FROM_CART,
+  SUM_TOTAL_PRICE,
+  ADD_ONE_MORE_ITEM,
+  REMOVE_ITEM,
+  QUANTITY_IN_SINGLE_PRODUCT,
+  INCREASE_MODIFY_QUANTITY,
+  DECREASE_MODIFY_QUANTITY,
+} from "./actionsTypes";
 
-export const currentBoardIDAction = (data) => {
+export const addProductToCart = (data) => {
   return {
-    type: FIRST_TYPE,
+    type: ADD_TO_CART,
     payload: data,
   };
 };
+
+const handleQuantitySubtraction = (data) => {
+  const copyOfData = data.slice();
+  for (let i = 0; i < copyOfData.length; i++) {
+    const currentItemId = data[i].id;
+    if (copyOfData[i].id === currentItemId) {
+      copyOfData[i].quantity--;
+    }
+  }
+  return copyOfData;
+};
+
+const handleQuantityAddition = (data) => {
+  const copyOfData = data.slice();
+  for (let i = 0; i < copyOfData.length; i++) {
+    const currentItemId = data[i].id;
+    if (copyOfData[i].id === currentItemId) {
+      copyOfData[i].quantity++;
+    }
+  }
+  return copyOfData;
```

Comment:
is this the quantity in the cart or the total quantity of stock you have? please add a comment above every single action explaining what it does